pymc3/model.py

```diff
@@ -914,7 +938,7 @@ def __init__(self, type=None, owner=None, index=None, name=None, data=None,
             data = pandas_to_array(data)
             type = TensorType(distribution.dtype, data.shape)
 
-        super(TensorVariable, self).__init__(type, None, None, name)
+        super(ObservedRV, self).__init__(type, owner, index, name)
```

Comment:
Stop, that's ok. `super` is right for that we call it on our class, not on base classes